spec/factories/profile.rb

```diff
@@ -2,5 +2,13 @@
   factory :profile do
     sequence(:name) { |n| "Profile name #{n}" }
     user
+
+    trait :filled_with_content do
+      after(:create) do |profile|
+        [*1..5].sample.times do
```

Comment:
I would recommend using a fixed number of sections here. In the longer period, you or another developer can write some spaces and has WTF? why sometimes I have one section, but another time I have more of them. 

Our goal is to write specs that always behave the same way. Adding this kind of randomness can cause randomly failing specs in long-term

BTW very nice use of `[*1..5]`, i didn't know that way of conversion range to array 👍